pkgs/by-name/cr/cryptpad/package.nix

```diff
@@ -121,7 +130,11 @@ buildNpmPackage {
     mkdir -p "$out_cryptpad/www/common/onlyoffice/dist"
     ${lib.concatMapStringsSep "\n" onlyoffice_install onlyoffice_versions}
     ${x2t_install}
-    rdfind -makehardlinks true -makeresultsfile false "$out_cryptpad/www/common/onlyoffice/dist"
+    # Run upstream's `install-onlyoffice.sh` script in `--check-only` mode to
+    # verify that we've installed the correct versions of the various
+    # OnlyOffice components.
+    patchShebangs --build $out_cryptpad/install-onlyoffice.sh
```

Comment:
(should that be in a separate `postPatch`? more of a theoretical problem than anything else, but it's handy to try things manually with `nix develop` occasionally)

Reply:
I don't think I can? IIUC, I'm patching a file that's created by the `npm run install:components` command we ran earlier in the `postInstall` that we're inside of.

Reply:
ah right it's in `$out_cryptpad`, I hadn't noticed that. Thanks!